src/main/kotlin/org/rust/ide/inspections/RsFieldInitShorthandInspection.kt

```diff
@@ -28,12 +28,16 @@ class RsFieldInitShorthandInspection : RsLocalInspectionTool() {
                     override fun getFamilyName(): String = "Use initialization shorthand"
 
                     override fun applyFix(project: Project, descriptor: ProblemDescriptor) {
-                        val field = descriptor.psiElement as RsStructLiteralField
-                        field.expr?.delete()
-                        field.colon?.delete()
+                        applyFix(descriptor.psiElement as RsStructLiteralField)
                     }
                 }
             )
         }
     }
+
+    fun applyFix(field: RsStructLiteralField)
```

Comment:
Seems to simple to extract. It's ok to copy-paste these 2 lines. Or at least let's move it to static method (with better name) instead of object method

Reply:
Moved it to a companion object and renamed to `applyShorthandInit`.